Buildings/Controls/OBC/ASHRAE/G36_PR1/AHUs/SingleZone/Economizers/Controller.mo

```diff
@@ -90,7 +90,7 @@ model Controller "Single zone VAV AHU economizer control sequence"
   parameter Modelica.SIunits.VolumeFlowRate VOutDes_flow=2.0
     "Calculated design outdoor airflow rate"
     annotation(Evaluate=true, Dialog(tab="Commissioning", group="Damper position limits"));
-  parameter Real minVOutMinFansSpePos(
```

Comment:
Please change the parameter names back to avoid non-backward compatible changes.